src/format/svg/SVGRenderer.hx

```diff
@@ -29,6 +29,7 @@ typedef ObjectFilter = String->GroupPath->Bool;
 
 class SVGRenderer
 {
+    public static inline var SQRT2:Float = 1.41421356237;

```

Comment:
Ah, `Math.sqrt(2)` isn't considered constant, so can't be in a `static inline` :/